src/index.html

```diff
@@ -12,21 +12,22 @@
   </head>
   <body>
     <div class="container flex-h">
-      <div class="list flex-h">
+      <div class="listWrapper flex-h">
         <div class="title flex-space-between">
           <h2 class="margin">Today's To Do</h2>
-          <i class="fa fa-refresh fa-x"></i>
+          <i class="fa fa-refresh fa-x margin"></i>
         </div>
-        <div class="flex-space-between">
+        <form class="newTask flex-space-between">
           <input
-            class="text"
+            class="newTaskName text"
             placeholder="Add to your list"
             type="text"
             name=""
             id="newTask"
```

Comment:
- [ ] New entries **do not match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal 5).

**Index mismatch in your application:**
![image](https://user-images.githubusercontent.com/12745474/135546974-d6b30d2e-eb17-4822-b021-fc096b633c93.png)

Reply:
thanks for mentioning that.